packages/components/src/mobile/bottom-sheet/test/cell.native.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render } from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import BottomSheetCell from '../cell.native.js';
+
+describe( '<BottomSheetCell />', () => {
+	it( 'renders the component', async () => {
+		const screen = render( <BottomSheetCell /> );
+		const cellTouchable = screen.getByTestId( 'cell-touchable' );
+
+		expect( cellTouchable ).toBeTruthy();
+	} );
+
+	it( 'displays a clear button when the text has value', async () => {
+		const onChangeValueMock = jest.fn();
+		const testValueString = 'test value string';
+		const newTestValueString = 'new test value string';
+
+		const screen = render(
+			<BottomSheetCell
+				displayClearButton={ true }
+				value={ testValueString }
+				onChangeValue={ onChangeValueMock }
+			/>
+		);
+
+		// Invoke editing behavior by pressing the Cell
+		const cellTouchable = await screen.getByTestId( 'cell-touchable' );
+		fireEvent.press( cellTouchable );
+
+		// Simulate onChangeValue events (without rerender)
+		const textInput = await screen.getByTestId( 'text-input' );
+		fireEvent( textInput, 'focus' );
+		fireEvent.changeText( textInput, newTestValueString );
+		expect( onChangeValueMock ).toHaveBeenCalledWith( newTestValueString );
```

Comment:
I understand that this test verifies that the clear button is displayed when it has text, however, I see that it also changes the text, is it really required? I'm wondering if it would be worth it to split it into two tests:
1. Check that the clear button is displayed when the text has value.
2. Check that the clear button remains displayed when the text has value and changes. [Although this seems a bit redundant, wdyt?]

Reply:
I agree with your reasoning here. Initially when I wrote these few lines of this test, I was expecting to use `rerender`, but I agree that this is redundant under a scoped component approach so I've removed these lines.

Reply:
Thanks for updating the tests with this suggestion 🙇 .